grpc-ballerina/tests/47_unary_timestamp_client.bal

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/time;
+
+@test:Config {enable: true}
+isolated function testUnaryTimestampWithGreeting() returns error? {
+    UnaryTimestampServiceClient utsClient = check new ("http://localhost:9147");
+    Greeting greeting = check utsClient->getGreeting("Hello");
+    [int, decimal] & readonly time = [1196676930, 0.12];
```

Comment:
Shall we replace `[int, decimal] & readonly` with `time:Utc`?

Reply:
fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/312/commits/7ae50091a63cc9c98fe07cbc4284b9134d2a06d5